src/newSeed/stage1.ts

```diff
@@ -1,5 +1,33 @@
-import { BaseStage } from "newSeed/baseStage";
-
+import { BaseStage } from "./baseStage";
+// Let's import the utils helper
+import {Utils} from "../services/utils";
 export class Stage1 extends BaseStage {
+  proceed(): void {
+    let message: string;
 
+    if (!this.seedConfig?.general?.projectName) {
+      message = "Enter a value for Project Name";
+    }
+    else if (!this.seedConfig?.general?.projectWebsite) {
+      message = "Plese enter the projec website";
+    } else if (!Utils.isValidUrl(this.seedConfig?.general?.projectWebsite)) {
+      message = "Please enter a valid website url";
+    }
+    else if (!this.seedConfig?.general?.category) {
+      message = "Enter a category";
+    } else if (!this.seedConfig?.general?.whitepaper) {
+      message = "We need your whitepaper address";
+    } else if (!Utils.isValidUrl(this.seedConfig?.general?.whitepaper)) {
+      message = "Please enter a valid website url";
+    } else if (!this.seedConfig?.general?.github) {
```

Comment:
delete this check and replace by padding `false` as the 2nd param to isValidUrl

Reply:
Delete the check ? or just add a false as a second param ?

Reply:
Delete the check.  note what false does

Reply:
don't forget to update the error message